sklearn/cluster/_dbscan_inner.pyx

```diff
@@ -26,26 +26,21 @@ def dbscan_inner(np.ndarray[np.uint8_t, ndim=1, mode='c'] is_core,
     cdef vector[np.npy_intp] stack
 
     for i in range(labels.shape[0]):
-        if labels[i] != -1 or not is_core[i]:
-            continue
```

Comment:
Please revert this. It's better not too indent the whole block under this `if` condition. It would reduce the diff and make this PR easier to review.